docs/user/dashboard/lens.asciidoc

```diff
@@ -19,301 +25,316 @@ data-type="inline"
-Each summary displays the percentage of sampled documents over all available documents.
+Change the fields list to display a different index pattern, different time range, or add your own fields.
 
-To view a data summary, click *i* next to the field.
+* To create a visualization with fields in a different index pattern, open the *Change index pattern* dropdown, then select the index pattern. 
 
+* If the fields list is empty, change the <<set-time-filter,time filter>>.
+
+* To add fields, open the action menu (*...*) next to the *Change index pattern* dropdown, then select *Add field to index pattern*.
++
 [role="screenshot"]
-image::images/lens_data_info_documents.png[Data summary analyzed documents]
+image:images/runtime-field-menu.png[Dropdown menu located next to index pattern field with items for adding and managing fields, width=50%]
++
+For more information about adding fields to index patterns and examples, refer to <<runtime-fields>>.
 
-Each data type display the following summaries:
+[float]
+[[lens-formulas]]
+==== Use formulas to perform math
 
-* *Date* &mdash; Displays the time distribution.
+Formulas allow you to perform math on aggregated data. The most common formulas divide two values to produce a percent.
 
-* *String* &mdash; Displays the top 10 values.
+. In the layer pane, click a field.
 
-* *Numeric* &mdash; Displays the top 10 values and a value distribution. When *Lens* displays the top 10 values distribution, the percentage of *Other* values is also displayed. 
-For array value fields, the percentage distribution considers each value in the array as separate.
+. Click *Formula*, then enter the formula. 
++
+Filter ratio example:: Use `kql=''` to filter one set of documents, then compare the document set to other documents within the same grouping:
++
+See how the error rate changes over time:
++
+```
+count(kql='response.status_code > 400') / count()
+```
++
+Week over week example:: Use `shift='1w'` to get the value of each grouping from the previous week. Time shift should not be used with the *Top values* function.
++
+```
+percentile(system.network.in.bytes, percentile=99) /
+percentile(system.network.in.bytes, percentile=99, shift='1w')
+```
++
+Percent of total example:: Formulas can calculate `overall_sum` for all the groupings,
+which lets you convert each grouping into a percent of total:
++
+```
+sum(products.base_price) / overall_sum(sum(products.base_price))
+```
++
+TIP: For detailed information on formulas, click image:dashboard/images/formula_reference.png[Formula reference icon].
 
-TIP: *Other* can equal more than 100% by a small amount.
+. To accurately display the formula, select *Percent* from the *Value format* dropdown.
 
 [float]
-[[create-the-visualization-panel]]
-==== Create visualizations
+[[compare-data-with-time-offsets]]
+==== Compare differences over time
```

Comment:
This section seems pretty thin to me- it's explaining that the feature _exists_ and how to find it, but not much more. I'm not sure what specific changes to suggest, but maybe it needs a second look.